tests/unit/amo/pages/TestLandingPage.js

```diff
@@ -53,6 +55,7 @@ describe(__filename, () => {
     errorHandler = createStubErrorHandler(),
     ...otherParams
   } = {}) => {
+    store.dispatch(setLang('en-US'));
```

Comment:
Why is this needed?

Reply:
Hmm, I guess this is another case of me adding this prematurely. I thought I needed it to get tests to pass, but apparently not. I'll remove it.